### PR TITLE
Add HTML/CSS, misc books, check_urls=free-programming-books-id.md

### PR DESCRIPTION
**Pull request by kadhirash**
## What does this PR do?
- Add resources | Cleans PR

### Description 
- Fixes #4323 
  - Add HTML book
  - Add Misc section for other collections of ebooks
- Add MySQL section at top since missing
### Why is this valuable (or not)?
- Adds books; cleans PR; improves repo
### How do we know it's really free?
- First resource is from blog post that includes author's free ebook; second resource is a collections of free ebooks
### For book lists, is it a book? For course lists, is it a course? etc.
- Books
## Checklist:
- [x] Read our [contributing guidelines](/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists  in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: free-programming-books-id.md [hunk 1]
```diff
@@ -66,6 +68,7 @@
 
 * [Belajar HTML dan CSS](http://www.ariona.net/ebook-belajar-html-dan-css/)
 * [Ebook Belajar HTML Dan CSS Dasar](https://www.malasngoding.com/download-ebook-belajar-html-dan-css-dasar-gratis/)
+* [Ebook HTML Dasar Untuk Pemula Lengkap](https://kodingbagus.blogspot.com/2018/07/ebook-html-dasar-untuk-pemula-lengkap.html) - Rio Astamal
```
Comment (eshellman) [inline free-programming-books-id.md:71]: not excited about a 15 year old book on html.

File: free-programming-books-id.md [hunk 2]
```diff
@@ -84,6 +87,11 @@
 * [Tutorial Dasar Javascript untuk Pemula](https://www.petanikode.com/tutorial/javascript/) *(Dalam Proses)*
 
 
+### Misc
+
+* [Kumplan Ebook Dari Berbagai Sumber](https://gilacoding.com/ebook) - Gila Coding
```
Comment (eshellman) [inline free-programming-books-id.md:92]: It's a list of books taken form other sites, including books already in our list.

**Review by eshellman — COMMENTED** [review]
going to pass on this one; except we're missing MySQL? maybe we moved it to SQL?

**Comment by kadhirash** [comment]
@eshellman yeah it varies from different files, sometimes it's under SQL and other times it's separated by itself. 

However in this file it's not at the top at all, but I'll see if I can add it in later today or after the other PR's.

**Pull request closed without merging**